documentation/src/main/asciidoc/stories/assembly_changes.adoc

```diff
@@ -3,7 +3,7 @@
 = {brandname} Version Details
 Learn about changes in {brandname} versions before you upgrade.
 
-include::{topics}/upgrading.adoc[leveloffset=+0]
+include::{topics}/changes.adoc[leveloffset=+0]
```

Comment:
Nitpick, but I don't think changing this file from `upgrading` to `changes` has any benefit in terms of being more descriptive, but it does complicate the git history.